toolchain/check/import_ref.cpp

```diff
@@ -2475,33 +2475,32 @@ static auto TryResolveTypedInst(ImportRefResolver& resolver,
     return ResolveResult::Retry();
   }
 
-  llvm::SmallVector<SemIR::FacetTypeInfo::ImplsConstraint> extend_constraints;
+  SemIR::FacetTypeInfo new_info = {.other_requirements =
```

Comment:
```suggestion
  SemIR::FacetTypeInfo local_facet_type_info = {.other_requirements =
```

(maybe consider renaming the other one to `import_facet_type_info`)